### ISSUE TITLE
So just because the global setting is bad, we are deprived of the annotation too? (aka: We want @FieldDefaults)

### ISSUE THREAD

**Issue by mberdev** [projectlombok/lombok#3686]
**Describe the feature**
My company has a policy not to use Lombok annotations _that show signs of being stuck in Experimental state._ This prevents us from using `@FieldDefaults` which is in that situation.

We simply want `@FieldDefaults` to come out of experimental, i.e. we want it to become a real feature.

The reason why it cannot come out of experimental (according to https://projectlombok.org/features/experimental/FieldDefaults ) seems absurd : It's because setting `lombok.fieldDefaults` which works on the **entire code base** is too dangerous.
But we don't care about `lombok.fieldDefaults` (i.e. we don't care about the **global** setting). Let us use `@FieldDefaults` locally!

In conclusion : separate the annotation from the global setting, and let `@FieldDefaults` become a proper feature.
